# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/mepo/compare.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

Strictly posix shells (ex: dash) don't recognize "==" to compare numbers.

  - NoTag <notag@maneto.fr>

--- mepo-1.1.2/scripts/mepo_ui_menu_user_pin_updater.sh.orig	2023-02-02 23:49:08.953100440 +0100
+++ mepo-1.1.2/scripts/mepo_ui_menu_user_pin_updater.sh	2023-02-02 23:49:24.932101320 +0100
@@ -72,7 +72,7 @@
 
 droppin() {
   SOURCEANDCOORDS="$(getcoords)"
-  [ $? == 1 ] && return
+  [ $? -eq 1 ] && return
   SOURCE="$(echo "$SOURCEANDCOORDS" | cut -d ' ' -f1)"
   COORDS="$(echo "$SOURCEANDCOORDS" | cut -d ' ' -f2-3)"
   LAT="$(echo $SOURCE | cut -d " " -f1)"
